networking/using-rfhe.adoc

```diff
@@ -0,0 +1,37 @@
+:_content-type: ASSEMBLY
+[id="using-rfhe"]
+= Monitoring Redfish hardware events
+include::_attributes/common-attributes.adoc[]
+:context: using-rfhe
+
+toc::[]
+
+:FeatureName: Redfish hardware events
+include::snippets/technology-preview.adoc[leveloffset=+1]
+
+[id="installing-hw-events"]
+== Installing the Baremetal Hardware Event Proxy Operator
+
+Install the Baremetal Hardware Event Proxy Operator using the {product-title} CLI or the web console.
+
+include::modules/nw-rfhe-installing-operator-cli.adoc[leveloffset=+2]
+
+include::modules/nw-rfhe-installing-operator-web-console.adoc[leveloffset=+2]
+
+include::modules/hw-installing-amq-interconnect-messaging-bus.adoc[leveloffset=+1]
+
+[id="subscribing-hw-events"]
+== Subscribing to Redfish BMC hardware events for a cluster node
+
+As a cluster administrator, you can subscribe to Redfish BMC events generated on a node in your cluster by creating a `BMCEventSubscription` custom resource (CR) for the node, and then creating a `HardwareEvent` CR for the event.
+
+include::modules/nw-rfhe-creating-hardware-event.adoc[leveloffset=+2]
+
+include::modules/cnf-rfhe-notifications-api-refererence.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
```

Comment:
In preview the "Additional resources" looks like a part of `api/cloudNotifications/v1/health/`. I suggest level up this title, or move it into `cnf-rfhe-notifications-api-refererence.adoc` and right before `The following API endpoints are available:`.